content/blog/translating-rmd.md

```diff
@@ -0,0 +1,132 @@
+---
+author: "Rob Ashton"
+date: 2021-06-30
+title: Translating R Markdown
+best: false
+tags:
+ - R
+ - rmarkdown
+ - translation
+---
+
+## Translating Rmd
+
+As part of our work on [naomi](../../projects/naomi) we've been asked to produce reports from R which can be distributed to country teams across Sub-Saharan Africa in English, French and Portuguese. R Markdown has no prescribed method for translating documents, but there are many options available to make this possible. This post covers 3 possible approaches: code chunks, custom blocks and tabsets.
+
+## Code chunks
+
+The first method uses conditionally evaluated [code chunks](https://bookdown.org/yihui/rmarkdown/r-code.html). We can use R Markdown chunk options `include` and `eval` to hide ouput and stop a chunk from being evaluated. Combining with `results="asis"` this allows us to output text conditionally based on the `lang` parameter.
+
+````r
+---
+title: '&nbsp;'
```

Comment:
I'm guessing this is just because the title is required? Even so, it's slightly cryptic - maybe a dummy title would avoid any confusion.

Reply:
Agreed. Also can the title be translated with this approach?